# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/time/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- time-1.9/src/resuse.c.vanilla	2023-05-12 23:12:20.339044647 +0200
+++ time-1.9/src/resuse.c	2023-05-12 23:12:34.614045678 +0200
@@ -25,6 +25,7 @@
 #include <sys/time.h>
 #include <sys/wait.h>
 #include <sys/resource.h>
+#include <string.h>
 
 #if !HAVE_WAIT3
 # include <sys/times.h>
